src/main/java/org/folio/ldp/TableObjController.java

```diff
@@ -39,7 +39,6 @@ public List<TableObj> getTableObjs() {
     return tableList; 
   }
 
-  @Cacheable(cacheNames="tableMap")
```

Comment:
Ah, so this is nothing to do with the Spring Boot upgrade -- it's fixing UILDP-65, right?

If so, we should move that issue into MODLDP, and you can reference it in the change-log entry.